mlflow/store/sqlalchemy_store.py

```diff
@@ -404,13 +405,21 @@ def delete_run(self, run_id):
 
     def log_metric(self, run_id, metric):
         _validate_metric(metric.key, metric.value, metric.timestamp, metric.step)
+        is_nan = math.isnan(metric.value)
+        if is_nan:
+            value = 0
+        elif not math.isfinite(metric.value):
```

Comment:
Does not work in Python2:
```
>>> math.isfinite
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'isfinite'
```

(Hopefully tests catch this)

Reply:
ah, nice catch

Reply:
Nice catch indeed, I had to fix this on my local MLflow since I like python2. I think you can just do 
```
elif math.isinf(metric.value)
```

Reply:
Nice, thanks